libusb/driver.c

```diff
@@ -548,6 +552,7 @@ int usb_modeswitchHandle(usb_driver_t *drv, usb_devinfo_t *dev, const usb_modesw
 }
 
 
+#ifndef USB_INTERNAL_ONLY
```

Comment:
Is there a way to minimize the use of `#ifdef`? They make code harder to follow. Do they really save code/data space? (ifdefing `ustack` saves memory for sure, but what about the rest?)